dcs/core/data_repository.py

```diff
@@ -165,19 +97,23 @@ def _get_model_with_self_uri(
         )
 
     async def _get_access_model(
-        self, *, drs_object: models.DrsObject, access_url: str
+        self, *, drs_object: models.DrsObject
     ) -> models.DrsObjectWithAccess:
         """Get a DRS Object model with access information."""
 
+        access_url = await self._object_storage.get_object_download_url(
+            bucket_id=self._config.outbox_bucket,
+            object_id=drs_object.file_id,
+            expires_after=30,
```

Comment:
We could think about setting this somewhere more global, e.g. in the config yaml, in case we want to change it later.